### DISCUSSION TITLE
Use of ChatGPT’s training process for other models

### DISCUSSION THREAD

**Discussion by qwertystars** [hpcaitech/ColossalAI#3068] [Community | Q&A]
Can I use other pre trained model like RoBERTa but train it using ChatGPT’s training method using Colossal Ai?

**Comment by Camille7777** [comment]
Hi @qwertystars Yes, for pre-trained model RoBERTa, we suggest to use the pretrained model from HuggingFace, you can find RoBERTa from [here](https://huggingface.co/docs/transformers/model_doc/roberta). Our example here #3223
